pkg/rtc/room.go

```diff
@@ -1155,3 +1127,86 @@ func (r *Room) DebugInfo() map[string]interface{} {
 
 	return info
 }
+
+func BroadcastDataPacketForRoom(r types.Room, source types.LocalParticipant, dp *livekit.DataPacket, logger logger.Logger) {
+	dest := dp.GetUser().GetDestinationSids()
+	var dpData []byte
+
+	participants := r.GetLocalParticipants()
```

Comment:
Can we just use `GetParticipants` here? There is no distinction of local/remote here.